frontend/src/app/modules/group/group/group.component.html

```diff
@@ -13,12 +13,12 @@
       </span>
       <h1 class="group-name">
         /
-        <a [routerLink]="">{{ group.name }} </a>
+        <a [routerLink]="">{{ group.name }}</a>
       </h1>
     </div>
     <div class="access-icon">
-      <i *ngIf="!group.isPublic" class="fas fa-lock"></i>
-      <i *ngIf="group.isPublic" class="fas fa-globe"></i>
+      <i *ngIf="group.isPublic.toString() === 'true'; else elseBlock" class="fas fa-globe"></i>
```

Comment:
Change to *ngIf="group.isPublic; else elseBlock"

Reply:
We get this value from the radio buttons which can't have boolean values, in other words, I need this check to convert the string value to boolean value. In the new changes, I moved this check to the group-settings page when we submit our changes.